corehq/apps/styleguide/templates/styleguide/bootstrap5/migration_guide.html

```diff
@@ -265,6 +282,72 @@ <h3 id="verify-references" class="pt-3">
     opening the Pull Request.
   </p>
 
+  <h3 id="re-migrate" class="pt-3">
+    Step 2.4: Re-Migrate Files
+  </h3>
+  <p>
+    There is a chance that during the time your split-files PR was open, someone might have added a new template or
+    javascript file, or made a change to one of the <code>bootstrap3</code> versions of an already split file.
+    If you merge <code>master</code> into your split files branch and find that the diffs have diverged for a file,
+    please run the following command to re-migrate the <code>bootstrap3</code> version of that file.
+  </p>
+  <p>
+    If the file is a template, please run:
```

Comment:
It'd be a nice simplification to have a single `--file-name` argument and have the tool detect whether it's a js or html file.

Reply:
do you want me to include this in this PR or do a followup PR?

Reply:
either, doesn't need to be a priority

Reply:
ended up including the change :)